src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/common/convs_and_sums.cpp

```diff
@@ -119,8 +119,11 @@ class ConvsAndSums : virtual public SubgraphBaseStaticTest {
 
         auto result = std::make_shared<ov::op::v0::Result>(relu3);
         function = std::make_shared<ov::Model>(result, params, "SimpleNet");
-
+#if defined(OPENVINO_ARCH_ARM64) || defined(OPENVINO_ARCH_ARM)
+        abs_threshold = 5e-2;
```

Comment:
Should depend on FP16 inference precision, not the arch. Please fix here and below.

Reply:
Fixed